src/ui/components/ExperimentBrowserPage.tsx

```diff
@@ -42,6 +45,40 @@ const ExperimentBrowserPage: FC = () => {
     void fetchExperiments();
   }, [fetchExperiments]);
 
+  const handleEnroll = async (experimentId: string, branchSlug: string) => {
+    console.log(branchSlug);
+    if (branchSlug !== "") {
+      const recipe = experiments.find((exp) => exp.id === experimentId);
+      const branch = recipe?.branches?.find((br) => br.slug === branchSlug);
+      try {
+        const result = await browser.experiments.nimbus.forceEnroll(
+          recipe,
+          branch,
+        );
+        if (result) {
+          console.log("Enrollment successful");
+          alert("Enrollment successful");
```

Comment:
Eventually we should come up with a better error story. Like a toast notification. But that can wait on #24. I'll file a follow-up.